### ISSUE TITLE
The loop's condition feels off

### ISSUE THREAD

**Issue by djalilhebal** [artiebits/fake-git-history#42]
First of all, nice utility.

I just checked its source code and the loop's condition felt off.
The end result will be fine (always < max commits per day), but it seems as if it's "changing its mind"...
Imagine the following execution of `i < getRandomIntInclusive(0, 2)`:
1. First iteration: `0 < 1`
2. Second iteration: `1 < 2`
3. Third iteration: `2 < 0`

A simple edit should make the intention clearer:
```diff
-    for (let i = 0; i < getRandomIntInclusive(...commitsPerDay); i++) {
+    const commitsCount = getRandomIntInclusive(...commitsPerDay);
+    for (let i = 0; i < commitsCount; i++) {
```

**Comment by smcnary** [comment]
Is the list on the my-history file supposed to be 1 entry or all the entries? Because on mine it's just the last entry in the block.

**Comment by artiebits** [comment]
hi @djalilhebal, 

a good catch, because it's actually a bug. indeed, the `getRandomIntInclusive` is being called on every iteration of the for loop. feel free to open a PR and fix it!

**Comment by artiebits** [comment]
solved in #43
